packages/core-commands/src/site-editor-navigation-commands.js

```diff
@@ -422,4 +478,9 @@ export function useSiteEditorNavigationCommands() {
 		hook: getSiteEditorBasicNavigationCommands(),
 		context: 'site-editor',
 	} );
+	useCommandLoader( {
+		name: 'core/edit-site/global-styles-css',
+		hook: getGlobalStylesOpenCssCommands(),
+		context: 'site-editor',
+	} );
```

Comment:
```suggestion
	useCommandLoader( {
		name: 'core/edit-site/global-styles-css',
		hook: getGlobalStylesOpenCssCommands(),
	} );
```

I think we shouldn't show this command by default:

| This PR | trunk |
|--------|--------|
| <img width="434" height="287" alt="Image" src="https://github.com/user-attachments/assets/d618b5ac-b71e-4001-a13c-8a42d294867d" /> | <img width="427" height="288" alt="Image" src="https://github.com/user-attachments/assets/da5710a7-0864-4ea1-b5d9-fe5225fa3c35" /> |

Reply:
Oh right!
Updated ✅